docker/src/main/java/org/jclouds/docker/compute/options/DockerTemplateOptions.java

```diff
@@ -384,7 +400,7 @@ public static DockerTemplateOptions overrideLoginCredentials(LoginCredentials cr
       }
 
       /**
-       * @see TemplateOptions#blockUntilRunning
+       * @see TemplateOptions#blockUntilRunning(boolean)
        */
       public static DockerTemplateOptions blockUntilRunning(boolean blockUntilRunning) {
          DockerTemplateOptions options = new DockerTemplateOptions();
```

Comment:
The code is using the pattern found in other similar classes, see `SoftLayerTemplateOptions` and so on. If we're going to change this would it not make more sense to change _all_ of these in a separate pull request?